poll/public/js/poll.js

```diff
@@ -10,7 +10,35 @@ function PollUtil (runtime, element, pollType) {
         this.votedUrl = runtime.handlerUrl(element, 'student_voted');
         this.submit = $('input[type=button]', element);
         this.answers = $('input[type=radio]', element);
+
+        // Set up gettext in case it isn't available in the client runtime:
+        if (typeof gettext == "undefined") {
+            window.gettext = function gettext_stub(string) { return string; };
+            window.ngettext = function ngettext_stub(strA, strB, n) { return n == 1 ? strA : strB; };
+        }
+
+        // Make gettext available in Handlebars templates
+        Handlebars.registerHelper('i18n', function(str) {
+            return gettext(str);
+        });
+
+        // Make i18n_ngettext available in Handlebars templates
+        Handlebars.registerHelper('i18n_ngettext', function(single_str, plural_str, total) {
+            return ngettext(single_str, plural_str, total);
+        });
+
+        // Make interpolate available in Handlebars templates
+        Handlebars.registerHelper('interpolate', function(formatString, parameters) {
+            parameters = parameters.hash;
+            return formatString.replace(/{\w+}/g,
+                function(parameter) {
+                    var parameterName = parameter.slice(1,-1);
```

Comment:
@asadiqbal08 Nit: Missing space between arguments to `slice`.

Same comment applies to corresponding line (L36) in poll_edit.js.